CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.4.0
+
+### Known Issues
+
+### Features Added
+- Multi-key signing: Ethsigner is initialised with a directory containing a number of TOML metadata files, each of which describe a key which may be used for signing. Upon reception of a Transaction, Ethsigner loads the corresponding metadata file, and signs the Transaction with the key defined therein.
+- Relaxed definition of 'optional' when parsing eth_SendTransaction (empty string, null an "0x" are deemed a missing optional parameter).
+- CI moved from Jenkins to CircleCI
+- Updated to Web3j 4.5.5
+- Updated to JUnit 5
```

Comment:
Think we should add something about EthSigner now proxying all unhandled requests not just /, so Besu /login will now work.

Reply:
bugger - I wrote this once, had that, then deleted it, started over ... and forget that one :(